README.md

```diff
@@ -1 +1,230 @@
+- [Implementation](#implementation)
+- [Deployment](#deployment)
+- [Testing](#testing)
+
+## What you’ll build 
+
+In this guide, you will build a simple Ballerina service that generates an UUID each time, and you will deploy this service on AKS. 
+
+## Compatibility
+
+| Ballerina Language Version 
+| -------------------------- 
+| 0.982.0
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
+- [Docker](https://docs.docker.com/engine/installation/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Docker Hub Account](https://hub.docker.com/)
+- [Microsoft Azure Account](https://azure.microsoft.com/en-us/free/)
+
+### Optional requirements
+
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina))
+
+## Implementation
+
+As the first step, you can build a Ballerina service that gives an UUID as the output. You can simply create a file `uuid_service.bal` and add the following content to the service code.
+
+```ballerina
+import ballerina/http;
+import ballerina/system;
+
+endpoint http:Listener uuid_ep {
+    port:8080
+};
+
+@http:ServiceConfig {
+    basePath:"/"
+}
+service<http:Service> uuid_service bind uuid_ep {
+
+    @http:ResourceConfig {
+        path:"/"
+    }
+    gen_uuid(endpoint outboundEP, http:Request request) {
+        _ = outboundEP->respond(system:uuid());
+    }
+
+}
+```
+
+Now you can add the Kubernetes annotations that are required to generate the Kubernetes deployment artifacts. 
+
+```ballerina
+import ballerina/http;
+import ballerina/system;
+import ballerinax/kubernetes;
+import ballerinax/docker;
+
+@kubernetes:Service {
+    name:"uuid-gen", 
+    serviceType:"LoadBalancer",
+    port:80
+}
+endpoint http:Listener uuid_ep {
+    port:8080
+};
+
+@kubernetes:Deployment {
+    enableLiveness:true,
+    image:"<username>/uuid-gen:latest",
+    push:true,
+    username:"<username>",
+    password:"<password>"
+}
+@http:ServiceConfig {
+    basePath:"/"
+}
+service<http:Service> uuid_service bind uuid_ep {
+
+    @http:ResourceConfig {
+        path:"/"
+    }
+    gen_uuid(endpoint outboundEP, http:Request request) {
+        _ = outboundEP->respond(system:uuid());
+    }
+
+}
+```
+We will be building a docker image here, and publishing it to Docker Hub. This is required, since we cannot simply have the docker image in the local registry, and run the Kubernetes applicates in AKS, where it needs to have access to the docker image in a globally accessible location. For this, an image name should be given in the format $username/$image_name in the "image" property, and "username" and "password" properties needs to contain the Docker Hub account username and password respectively. The property "push" is set to "true" to signal the build process to push the build docker image to Docker Hub.
```

Comment:
Change "docker" to Titlecase, i.e., "Docker". Apply elsewhere in the guide if applicable. 

Remove the comma before "and publishing.." in the first line. You ideally wouldn't need a comma here.

Reply:
Done, thanks!.